main.go

```diff
@@ -707,6 +707,163 @@ func postUnlockPageHandler(c *gin.Context, db *sql.DB) {
 	c.Redirect(http.StatusFound, "http://localhost:8000/"+username+"/"+repoName+"/issues/"+IssueNumber)
 }
 
+type Collaborator struct {
+	Username  string
+	UserImage string
+	Name      string
+}
+
+func getCollaboratorPageHandler(c *gin.Context, db *sql.DB) {
+
+	currentUser, err := authorize(c, db)
+	authorized := err == nil
+
+	username := c.Param("user_name")
+	repoName := c.Param("repo_name")
+
+	IsRepoOwner := currentUser.Username == username
+	if !IsRepoOwner {
+		c.Redirect(http.StatusFound, "http://localhost:8000/"+currentUser.Username)
+		return
+	}
+
+	currentRepo, err := getCurrentRepo(db, username, repoName)
+	if err != nil {
+		fmt.Println(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	rows, err := db.Query(`SELECT users.username,users.image,users.name FROM users JOIN collaborators ON users.id = collaborators.user_id 
+								WHERE collaborators.repo_id = $1;`, currentRepo.RepoId)
+	if err != nil {
+		fmt.Println(err)
+		c.AbortWithStatus(http.StatusInternalServerError)
+		return
+	}
+
+	collaborators := []Collaborator{}
+
+	var UserName, image, name string
```

Comment:
move variable declaration inside the for loop. Since they are not used outside elsewhere.

Reply:
Done